fedimint-server/src/net/peers.rs

```diff
@@ -258,6 +258,7 @@ where
 {
     #[must_use]
     async fn send(&mut self, peers: &[PeerId], msg: T) -> Cancellable<()> {
+        trace!(target: LOG_NET_PEER, ?peers, "sending message to list of");
```

Comment:
If you intend to land it, the `to list of` could be skipped. Generally these `to`, `with` suffixes seem out of place. The message is "doing x" and doesn't need to refer to associated data.